excludes/id.php

```diff
@@ -107,17 +116,19 @@
     'Rwanda',
     'Saint Barthelemy',
     'Saint Helena',
-    'Saint Kitts And Nevis',
+    'Saint Kitts and Nevis',
     'Saint Lucia',
     'Saint Martin',
-    'Saint Pierre And Miquelon',
-    'Saint Vincent And Grenadines',
+    'Saint Pierre and Miquelon',
```

Comment:
The keys to exclude are specified here.

`'Saint Pierre And Miquelon' !== 'Saint Pierre and Miquelon'`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L444